app/templates/docs/trusted-publishing.hbs

```diff
@@ -0,0 +1,120 @@
+<PageHeader @title="Trusted Publishing" />
+
+<TextContent @boxed={{true}}>
+  <h2>What is Trusted Publishing?</h2>
+  <p>
+    Trusted Publishing is a secure way to publish your Rust crates from GitHub Actions without manually managing API tokens.
+    It uses OpenID Connect (OIDC) to verify that your workflow is running from your repository, then provides a short-lived token for publishing.
+  </p>
+
+  <p>
+    Instead of storing long-lived API tokens in your repository secrets, Trusted Publishing allows GitHub Actions to authenticate
+    directly with crates.io using cryptographically signed tokens that prove the workflow's identity.
+  </p>
+
+  <p>
+    <strong>Note:</strong> crates.io currently only supports GitHub Actions, but we are planning to support other
+    CI/CD platforms like GitLab CI/CD in the future.
+  </p>
+
+  <h3>Security Benefits</h3>
+  <ul>
+    <li><strong>No long-lived API tokens</strong> to manage or rotate</li>
+    <li><strong>Tokens automatically expire</strong> after 30 minutes</li>
+    <li><strong>Repository and workflow verification</strong> prevents unauthorized publishing</li>
+    <li><strong>OIDC-based cryptographic verification</strong> with GitHub's public JWKS</li>
+    <li><strong>Optional GitHub Actions environments</strong> for additional access controls</li>
+  </ul>
+
+  <h2>Quick Start</h2>
+  <p>Follow these steps to set up Trusted Publishing for your crate:</p>
+
+  <ol>
+    <li><strong>Configure your crate for Trusted Publishing</strong> in the crates.io settings</li>
+    <li><strong>Set up your GitHub Actions workflow</strong> with the required permissions and authentication action</li>
+    <li><strong>Publish your crate</strong> using the automated workflow</li>
+  </ol>
+
+  <h3>Prerequisites</h3>
+  <ul>
+    <li>Your crate must already be published to crates.io (initial publish requires an API token)</li>
+    <li>You must be an owner of the crate on crates.io</li>
+    <li>Your repository must be on GitHub</li>
+  </ul>
+
+  <h2>Configuring Trusted Publishing</h2>
+  <p>
+    Configure your crate on crates.io:
+  </p>
+
+  <ol>
+    <li>Go to your crate's Settings → Trusted Publishing</li>
+    <li>Click "Add Trusted Publisher" and fill in:
+      <ul>
+        <li><strong>Repository owner:</strong> Your GitHub username or organization</li>
+        <li><strong>Repository name:</strong> The name of your repository</li>
+        <li><strong>Workflow filename:</strong> The filename of your GitHub Actions workflow (e.g., "release.yml")</li>
+        <li><strong>Environment:</strong> Optional environment name if you're using GitHub environments</li>
+      </ul>
+    </li>
+    <li>Save the configuration</li>
+  </ol>
+
+  <h2>GitHub Actions Setup</h2>
+  <p>
+    Create a workflow file at <code>.github/workflows/release.yml</code>:
+  </p>
+
+  {{!-- template-lint-disable no-whitespace-for-layout  --}}
+  <pre><code>name: Publish to crates.io
+on:
+  push:
+    tags: ['v*']
```

Comment:
We should probably explain before this snippet that this will publish each time a `v` tag is pushed (since that may not be super obvious to someone who's not very familiar with GitHub Actions).

Reply:
<img width="955" alt="Bildschirmfoto 2025-06-23 um 19 46 57" src="https://github.com/user-attachments/assets/5f59f89e-78aa-4f02-a6aa-2ace936a0693" />

done!

I've also added syntax highlighting to the snippet to make it easier to digest.